libraries/standard/mqtt/include/iot_mqtt_helper.h

```diff
@@ -22,7 +22,7 @@
 
 /**
  * @file iot_mqtt_helper.c
- * @brief Implements helper functions for the MQTT library.
+ * @brief Implements internal helper functions for the MQTT library.
```

Comment:
Maybe rename this iot_mqtt_serialize_internal.h, as all the functions are for the serializer?

Also should be moved to src/private. include/ is for public headers only.

Reply:
I have moved it to private folder. I'll rename it in the next PR since by then we it will be clear which code resides where.